src/components/CryptoTraq.js

```diff
@@ -51,6 +67,24 @@ const CryptoTraq = (props) => {
 };
 
 //  Radium Styles
+/*
+		Added selectGlowAnim to add a small animation to the selection element to indicate to the user
+		that they should take action there first.  Applied with select__glow
+		I added this trackBtnAnimation to add a small scale animation to the track button as the
+		3rd step to tracking your conversion rate.  It gets passed in as props and set in the button component
+ */
+
```

Comment:
- Added `selectGlowAnim` keyframe to add a small animation to the selection element to indicate to the user that they should take action there first.  Applied with `select__glow` style.
- I added this `trackBtnAnimation` keyframe to add a small scale animation to the track button as the 3rd step to tracking your conversion rate.  It gets passed in as props and set in the button component.  Applied with `trackAnim` style.